src/br/com/harbitech/school/course/Course.java

```diff
@@ -28,44 +27,36 @@ public Course(String name, String codeUrl, int completionTimeInHours, String ins
 
         this.name = name;
         this.codeUrl = codeUrl;
-        this.completionTimeInHours = completionTimeInHours;
+        this.completionTimeInHours = -1;
```

Comment:
Acho que não entendi muito bem o -1 aqui, sendo que você recebe o parâmetro no construtor. Não poderia deixar da forma antiga, com o `this.completionTimeInHours = completionTimeInHours`?

Reply:
good catch :fishing_pole_and_fish: